internal/controllers/build_sign_reconciler.go

```diff
@@ -64,13 +64,6 @@ func NewBuildSignReconciler(
 	}
 }
 
-//+kubebuilder:rbac:groups=kmm.sigs.x-k8s.io,resources=modules,verbs=get;list;watch;update;patch
```

Comment:
All the changes needs to also be applied to `internal/controller/hub/managedclustermodule_recondiler.go`

Reply:
I take it back. Everything needed for KMM-hub is already present in the MCM controller...